### PR TITLE
arbitrary java currency

### PR DESCRIPTION
**Pull request by mikhailZakharevich**
added an implicit that picks one element from java Currency

### CODE REVIEW

**Comment by VLADISLAV008** [comment]
Why is this Pull Request open into the `main` branch?

**Comment by mikhailZakharevich** [comment]
Oops, overlooked this part

File: src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/CurrencyArbitraries.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package group.scala.karazin.circe.literal.extras.arbitraries
+
+import cats.implicits._
+import io.circe.syntax._
```
Comment (IgorWolkov) [inline src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/CurrencyArbitraries.scala]: Is there any unnecessary imports?

**Review by IgorWolkov — CHANGES_REQUESTED** [review]
Please cleanup imports if possible.

File: src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/CurrencyArbitraries.scala (hunk 1 above)
Comment (mikhailZakharevich) [reply src/test/scala/group/scala/karazin/circe/literal/extras/arbitraries/CurrencyArbitraries.scala]: Done

**Review by IgorWolkov — APPROVED** [review]

**Pull request merged**